styles.css

```
@@ -0,0 +1,120 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+html {
+  padding: 0;
+  margin: 0;
+}
+
+#container {
+  /* width: 375px;
+  height: 710px; */
+  border: 1px solid #e5e5e5;
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+
+  /* margin: 0 auto; */
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+  height: 90px;
+
+  /* margin-top: 55px; */
+  background-color: #fff;
+  padding: 55px 25px 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+header span {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+nav .bar1,
+nav .bar2,
+nav .bar3 {
+  width: 20px;
+  height: 2px;
+  background-color: #6070ff;
+  margin-bottom: 5px;
+}
+
+.headline {
+  background: #fff url("images/header-shapes-mobile.png");
+
+  /* background-size: cover; */
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  border-bottom-left-radius: 10%;
+  padding: 55px 20px 0;
+}
```

- [ ] Hello, you did an excellent job of making changes based on the previous reviewer's suggestions :clap: . However, when compared to the template, the background image is still not displaying as expected. Please ensure to fix that. :ok_hand: 😊

To help you I would suggest you add `background-size: contain;` and ` background-repeat: no-repeat;` to the `headline` class.
```suggestion
.headline {
  background: #fff url("images/header-shapes-mobile.png");
  background-repeat: no-repeat;
  background-size: contain;
  display: flex;
  flex-direction: column;
  height: 100%;
  border-bottom-left-radius: 10%;
  padding: 55px 20px 0;
}
```